src/components/SearchResult/SearchResult.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './SearchResult.css';
+import { Link } from "react-router-dom";
+
+const SearchResult = ({img, name, id, brand, tags, productType}) => {
+    const styledTags = tags.join(', ');
+    return (
+        <Link to={`/:id`}>
```

Comment:
Maybe a way to prevent the routing troubles is make the Route more specific? I had a situation that occurred in RT that needed us to have a more specific url to allow us to accurately route.